src/riak_kv_wm_json.erl

```diff
@@ -90,9 +152,9 @@ standards. The decoder is tested using [JSONTestSuite](https://github.com/nst/JS
 
 -type encoder() :: fun((dynamic(), encoder()) -> iodata()).
 
--doc """
-Simple JSON value encodeable with `json:encode/1`.
-""".
+% -doc """
```

Comment:
Since you commented this out (possibly because it refers to `json:` you may want to actually remove it?